build-Error on Windows, Versions > 3.3.4

I try to install current or some older version of png-img using the following commands:
```
npm i -g png-img@3.3.6
npm i -g png-img
```
But i got errors see below.

Install the version 3.3.4 works without problems.
```
npm i -g png-img@3.3.4
```
I suspect it has something to do with ARM code. I am using windows 11, X64, msvs2022 and node-gyp@9.1.0

```
PS C:\Users\dennh> npm i -g png-img@3.3.6
npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.
npm ERR! code 1
npm ERR! path C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c C:\Users\dennh\AppData\Local\Temp\install662547423.cmd
npm ERR! > png-img@3.3.6 build-release
npm ERR! > node-gyp rebuild --release
npm ERR!
npm ERR! Die Projekte in dieser Projektmappe werden nacheinander erstellt. Um eine parallele Erstellung zu erm�glichen, m�ssen Sie den Schalter "-m" hinzuf�gen.
npm ERR!   Assembling ..\..\..\third_party\libpng\arm\filter_neon.S...
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(2): error A2008: syntax error : / [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(3): error A2008: syntax error : * [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(4): error A2008: syntax error : * [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(5): error A2008: syntax error : * [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(6): error A2008: syntax error : * [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(7): error A2008: syntax error : * [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(8): error A2008: syntax error : * [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(9): error A2008: syntax error : * [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(10): error A2008: syntax error : and [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(11): error A2008: syntax error : * [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(13): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(14): error A2008: syntax error : definitions [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(15): error A2008: syntax error : * [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(16): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(17): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(18): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(19): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(20): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(21): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(22): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(23): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(24): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(25): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(26): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(27): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(28): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(29): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(30): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(31): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(32): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(33): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(34): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(35): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(36): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(37): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(38): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(39): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(40): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(41): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(42): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(43): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(44): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(45): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(46): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(47): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(48): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(49): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(50): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(51): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(52): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(53): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(54): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(55): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(56): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(57): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(58): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(59): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(60): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(61): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(62): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(63): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(64): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(65): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(66): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(67): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(68): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(69): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(70): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(71): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(72): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(73): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(74): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(75): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(76): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(77): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(78): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(79): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(80): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(81): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(82): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(83): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(84): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(85): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(86): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(87): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(88): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(89): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(90): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(91): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(92): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(93): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(94): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(95): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(96): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(97): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(98): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(99): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(100): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(101): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(102): error A2044: invalid character in file [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! ..\..\..\third_party\libpng\arm\filter_neon.S(103): fatal error A1012: error count exceeds 100; stopping assembly [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR! C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\MSBuild\Microsoft\VC\v170\BuildCustomizations\masm.targets(70,5): error MSB3721: Der Befehl "ml64.exe /c /nologo /Zi /Fo"Release\obj\libpng\filter_neon.obj" /W3 /errorReport:prompt  /Ta..\..\..\third_party\libpng\arm\filter_neon.S" wurde mit Code 1 beendet. [C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\third_party\libpng\libpng.vcxproj]
npm ERR!   adler32.c
npm ERR!   compress.c
npm ERR!   crc32.c
npm ERR!   deflate.c
npm ERR!   gzclose.c
npm ERR!   gzlib.c
npm ERR!   gzread.c
npm ERR!   gzwrite.c
npm ERR!   infback.c
npm ERR!   inffast.c
npm ERR!   inflate.c
npm ERR!   inftrees.c
npm ERR!   trees.c
npm ERR!   uncompr.c
npm ERR!   zutil.c
npm ERR!   win_delay_load_hook.cc
npm ERR!   zlib.vcxproj -> C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img\build\Release\\zlib.lib
npm ERR! prebuild-install warn install No prebuilt binaries found (target=16.16.0 runtime=node arch=x64 libc= platform=win32)
npm ERR! npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.
npm ERR! npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@9.0.0
npm ERR! gyp info using node@16.16.0 | win32 | x64
npm ERR! gyp info find Python using Python version 3.10.5 found at "C:\Python310\python.exe"
npm ERR! gyp info find VS using VS2022 (17.3.32825.248) found at:
npm ERR! gyp info find VS "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools"
npm ERR! gyp info find VS run with --verbose for detailed information
npm ERR! gyp info spawn C:\Python310\python.exe
npm ERR! gyp info spawn args [
npm ERR! gyp info spawn args   'C:\\Users\\dennh\\AppData\\Roaming\\npm\\node_modules\\npm\\node_modules\\node-gyp\\gyp\\gyp_main.py',
npm ERR! gyp info spawn args   'binding.gyp',
npm ERR! gyp info spawn args   '-f',
npm ERR! gyp info spawn args   'msvs',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   'C:\\Users\\dennh\\AppData\\Roaming\\npm\\node_modules\\png-img\\build\\config.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   'C:\\Users\\dennh\\AppData\\Roaming\\npm\\node_modules\\npm\\node_modules\\node-gyp\\addon.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   'C:\\Users\\dennh\\AppData\\Local\\node-gyp\\Cache\\16.16.0\\include\\node\\common.gypi',
npm ERR! gyp info spawn args   '-Dlibrary=shared_library',
npm ERR! gyp info spawn args   '-Dvisibility=default',
npm ERR! gyp info spawn args   '-Dnode_root_dir=C:\\Users\\dennh\\AppData\\Local\\node-gyp\\Cache\\16.16.0',
npm ERR! gyp info spawn args   '-Dnode_gyp_dir=C:\\Users\\dennh\\AppData\\Roaming\\npm\\node_modules\\npm\\node_modules\\node-gyp',
npm ERR! gyp info spawn args   '-Dnode_lib_file=C:\\\\Users\\\\dennh\\\\AppData\\\\Local\\\\node-gyp\\\\Cache\\\\16.16.0\\\\<(target_arch)\\\\node.lib',
npm ERR! gyp info spawn args   '-Dmodule_root_dir=C:\\Users\\dennh\\AppData\\Roaming\\npm\\node_modules\\png-img',
npm ERR! gyp info spawn args   '-Dnode_engine=v8',
npm ERR! gyp info spawn args   '--depth=.',
npm ERR! gyp info spawn args   '--no-parallel',
npm ERR! gyp info spawn args   '--generator-output',
npm ERR! gyp info spawn args   'C:\\Users\\dennh\\AppData\\Roaming\\npm\\node_modules\\png-img\\build',
npm ERR! gyp info spawn args   '-Goutput_dir=.'
npm ERR! gyp info spawn args ]
npm ERR! gyp info spawn C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\MSBuild\Current\Bin\MSBuild.exe
npm ERR! gyp info spawn args [
npm ERR! gyp info spawn args   'build/binding.sln',
npm ERR! gyp info spawn args   '/clp:Verbosity=minimal',
npm ERR! gyp info spawn args   '/nologo',
npm ERR! gyp info spawn args   '/p:Configuration=Release;Platform=x64'
npm ERR! gyp info spawn args ]
npm ERR! gyp ERR! build error
npm ERR! gyp ERR! stack Error: `C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\MSBuild\Current\Bin\MSBuild.exe` failed with exit code: 1
npm ERR! gyp ERR! stack     at ChildProcess.onExit (C:\Users\dennh\AppData\Roaming\npm\node_modules\npm\node_modules\node-gyp\lib\build.js:194:23)
npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:527:28)
npm ERR! gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:291:12)
npm ERR! gyp ERR! System Windows_NT 10.0.22000
npm ERR! gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\dennh\\AppData\\Roaming\\npm\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild" "--release"
npm ERR! gyp ERR! cwd C:\Users\dennh\AppData\Roaming\npm\node_modules\png-img
npm ERR! gyp ERR! node -v v16.16.0
npm ERR! gyp ERR! node-gyp -v v9.0.0
npm ERR! gyp ERR! not ok

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\dennh\AppData\Local\npm-cache\_logs\2022-09-07T10_36_38_663Z-debug-0.log
```

--- a/package.json
+++ b/package.json
@@ -26,7 +26,7 @@
     "eslint": "^8.23.0",
     "eslint-config-gemini-testing": "^3.0.0",
     "mocha": "^2.0.1",
-    "node-gyp": "^6.1.0",
+    "node-gyp": "^9.1.0",
     "prebuild": "^11.0.0",
     "sinon": "^1.17.3",
     "typescript": "^4.5.3"

